src/screens/Modal/Scan/ScanModal.tsx

```diff
@@ -153,14 +155,67 @@ class ScanView extends Component<Props, State> {
             });
     };
 
+    addCustomNode = (hash: string) => {
+        this.setState({
+            isLoading: true,
+        });
+
+        BackendService.getEndpointDetails(hash)
+            .then((res: any) => {
+                const { endpoint } = res;
+                const { explorer, name, node } = endpoint;
+
+                // create endpoint base on backend response
+                const customNode = {
```

Comment:
@N3TC4T This is amazing work, and so useful to our team. Thank you!!

Dinner or drinks on us when we meet in person one day!

@nkramer44 @theotherian